lib/handler/mruby/embedded.c.h

```diff
@@ -79,6 +79,13 @@
     "  def sleep(*sec)\n"                                                                                                          \
     "    _h2o__sleep(*sec)\n"                                                                                                      \
     "  end\n"                                                                                                                      \
+    "  def task(&block)\n"                                                                                                         \
+    "    fiber = Fiber.new do\n"                                                                                                   \
+    "      block.call\n"                                                                                                           \
+    "      _h2o__finish_child_fiber()\n"                                                                                           \
```

Comment:
Would you mind elaborating what happens if `block.call` raises an exception? My expectation is that `h2o__finish_child_fiber()` will not be called in such case.

If calling `h2o__finish_child_fiber()` makes any difference, then I believe that we should call the function even if `block.call` raises an error. Or if `h2o__finish_child_fiber()` is a noop, can we eliminate the function?

Reply:
For now, entire application will stop when `block.call` raises.
Then I fixed to call `_h2o__send_error` and not to call `_h2o__finish_child_fiber` intentionally.
By this, task will raise an error and only task dies but parent application will not stop when fiber in task raises.
If I should make it parent application stop, I'll fix it.
https://github.com/h2o/h2o/pull/1336/commits/52ea22320fa0c178705b4b041495dcb4f3f85dde

Reply:
perfect! :D